cmd/nanotube/listen.go

```diff
@@ -97,32 +88,37 @@ loop:
 		select {
 		case <-term:
 			// stop accepting new connections on termination signal
-			break loop // need to break both select and the loop
+			err := l.Close()
+			if err != nil {
+				lg.Error("failed to close listening TCP connection", zap.Error(err))
+			}
+			break loop
 		case err := <-errCh:
-			lg.Error("accpeting connection failed", zap.Error(err))
+			lg.Error("accepting connection failed", zap.Error(err))
 		case conn := <-connCh:
 			wg.Add(1)
 			go readFromConnectionTCP(&wg, conn, queue, term, cfg, ms, lg)
 		}
 	}
-	lg.Info("Termination: stopped accepting new TCP connections. Starting to close incoming connections...")
+	lg.Info("Termination: Stopped accepting new TCP connections. Starting to close incoming connections...")
 	wg.Wait()
-	lg.Info("Termination: Finished incoming TCP connections...")
+	lg.Info("Termination: Finished previously accpted TCP connections.")
```

Comment:
Typo in "accpted"

Reply:
Will fix later because this is actually in #79